change/@microsoft-teams-js-59df1594-8bfb-454c-8223-13c6e8c1e73d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "appEntity APIs will now throw a NOT_SUPPORTED_ON_PLATFORM error if called while capability is not supported.",
```

Comment:
This should be consistent with the other isSupported change files: "Adding isSupported check to appEntity capability"